modin/pandas/test/test_general.py

```diff
@@ -18,7 +18,12 @@
 from numpy.testing import assert_array_equal
 from modin.utils import get_current_backend, to_pandas
 
-from .utils import test_data_values, test_data_keys, df_equals
+from .utils import (
```

Comment:
I like these changes to combine `sort_index_for_equal_values` into one place, but let's move it to a new PR.